camera/buildDetector.py

```diff
@@ -13,21 +13,21 @@ def addAmp(ampCatalog,i,rN,gain_s):
 
     #This needs to be the full dimension of what your amp outputs,
     #including overscan, any dummy pixels etc etc.:
-    width = 6666
-    height = 4454
+    width = 3333
+    height = 2227
 
-    os = 10 #pixels of overscan
+    os = 20   #pixels of overscan
 
     #This is the dimensions of the active region of your amp:
-    bbox = afwGeom.Box2I(afwGeom.Point2I(0, 0), afwGeom.Extent2I(6602, 4366))
+    bbox = afwGeom.Box2I(afwGeom.Point2I(0, 0), afwGeom.Extent2I(3288, 2192))
 
     #If your CCD consists of more than one amp, you'll need to
     #some of them (i.e., the one on the right is shifted by X pixels)
-    bbox.shift(afwGeom.Extent2I(6602*i,0))
+    bbox.shift(afwGeom.Extent2I(3288*1 if (i==1 or i==3) else 0, 2192*1 if (i==2 or i==3) else 0))
 
     #Define the gain, saturation and readout noise:
     gain = gain_s
-    saturation = 65535
+    saturation = 65535  ##not sure
     readNoise = rN
 
     #Which corner is the data read out from?
```

Comment:
The png diagram suggests amps 2 and 3 have UL and UR readouts, respectively. I don't know whether this is important, but I guess we should give it a go.